setup.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 # Create virtual environment
-python3 -m venv venv
+python3 -m venv .venv
```

Comment:
I made this change because it is a common practice to make a virtual environment directory hidden for the project. Same as followed by VSCode or maybe others too, so this will make the project more IDE-friendly and clean.